SemanticUI/Modules/Common.elm

```diff
@@ -0,0 +1,36 @@
+module SemanticUI.Modules.Common exposing (Render, WrappedContent, WrappedNode)
+
+import Html exposing (..)
+
+
+{-| Convenience alias for a HTML DOM Node render function
+-}
+type alias Render msg =
+    List (Attribute msg) -> List (Html msg) -> Html msg
+
+
+{-| Convenience type alias for an `Html.Node` wrapper to augment node behaviour
+
+It takes the following:
+
+  - Node type as a render function e.g. `Html.div`
+  - Extra attributes of node
+  - The children of node
+
+-}
+type alias WrappedNode msg =
+    Render msg -> List (Attribute msg) -> List (Html msg) -> Html msg
```

Comment:
I think this should be something like `WrappedElement`, because `Html.text` is also a DOM node, but we're only interested in nodes that take attrs and child nodes as arguments.

Reply:
Then again elm has an `Html.node` function, so I see where this comes from. But I think it's the html package that is mistaken in this case.

> An element is a specific type of node, one that can be directly specified in the HTML with an HTML tag and can have properties like an id or a class. can have children, etc... There are other types of nodes such as comment nodes, text nodes, etc... with different characteristics. Each node has a property .nodeType which reports what type of node it is.

From https://stackoverflow.com/a/9979779/167485